### PR TITLE
Delete Instance from UI

### PR DESCRIPTION
**Pull request by OferLevi85**
Signed-off-by: Ofer Levi <ofer.levi@verizonmedia.com>

### CODE REVIEW

File: ui/src/__tests__/components/domain/UserDomain.test.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ import { renderWithRedux } from '../../../tests_utils/ComponentsTestUtils';
 import MockApi from '../../../mock/MockApi';
```
Comment (OferLevi85) [inline ui/src/__tests__/components/domain/UserDomain.test.js:21]: No changes were made to this file - diffs are from running fix-lint.

File: ui/src/__tests__/components/group/GroupList.test.js [hunk 2]
```diff
@@ -17,7 +17,11 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import GroupList from '../../../components/group/GroupList';
 import API from '../../../api';
```
Comment (OferLevi85) [inline ui/src/__tests__/components/group/GroupList.test.js:19]: No changes were made to this file - diffs are from running fix-lint.

File: ui/src/__tests__/components/header/DomainDetails.test.js [hunk 3]
```diff
@@ -26,19 +26,18 @@ import MockApi from '../../../mock/MockApi';
 const mockApi = {
     getMeta: jest.fn().mockReturnValue(
         new Promise((resolve, reject) => {
-                resolve([])
-            }
-        )
+            resolve([]);
+        })
     ),
-}
```
Comment (OferLevi85) [inline ui/src/__tests__/components/header/DomainDetails.test.js:33]: No changes were made to this file - diffs are from running fix-lint.

**Comment by mendi160** [comment]
> Signed-off-by: Ofer Levi [ofer.levi@verizonmedia.com](mailto:ofer.levi@verizonmedia.com)

The redux part LGTM

**Review by mendi160 — APPROVED** [review]

**Pull request merged**